plugins/ms-vscode.node-debug/1.0.0/meta.yaml

```diff
@@ -0,0 +1,13 @@
+id: ms-vscode.node-debug
+version: 1.0.0
+type: VS Code extension
+name: Node Debug (legacy)
+title: Node.js debugging support (legasy)
```

Comment:
typo in word `legasy`

Reply:
@sleshchenko 
You can find every tiny bugs ;)
That's really cool.

Reply:
:eyes: :smile_cat: